### PR TITLE
Update privilege document according to the latest privilege info

### PR DESCRIPTION
**Pull request by yl33**
Update 5.5 privilege changes:
 - Add: autofillmanager, d2d.datasharing, d2d.remotelaunch, securesysteminfo, voicecontrol.tts and windowsystem.admin
 - Change: telephony (removed access permission to non-resettable secure device information such as IMEI)

### CODE REVIEW

File: docs/application/dotnet/tutorials/sec-privileges.md [hunk 1]
```diff
@@ -140,24 +143,27 @@ when using security-sensitive API modules in .net applications:
...
 | `http://tizen.org/privilege/vpnservice` | public |  | The application can manage the Virtual Private Network (VPN) and change its settings. |
 | `http://tizen.org/privilege/web-history.admin` | platform | User history | The application can manage user's internet history. |
 | `http://tizen.org/privilege/widget.viewer` | public |  | The application can show widgets, and information from their associated applications, on the home screen. |
 | `http://tizen.org/privilege/wifidirect` | public |  | The application can enable and disable Wi-Fi Direct, manage Wi-Fi Direct connections, and change Wi-Fi Direct settings. |
 | `http://tizen.org/privilege/window.priority.set` | public |  | The application can appear on top of other windows and screens, including the lock screen, according to the order of priority of the windows. This may prevent user from interacting with other applications or screens until the window for this application is closed. |
+| `http://tizen.org/privilege/windowsystem.admin` | platform | The application can change the settings for services provided by the display server, such as the quick panel and softkey bar. |
```
Comment (mijin85cho) [inline docs/application/dotnet/tutorials/sec-privileges.md]: The last cell should be moved to right.

image

File: docs/application/native/tutorials/details/sec-privileges.md [hunk 2]
```diff
@@ -145,24 +148,27 @@ when using security-sensitive API modules in native applications:
...
 | `http://tizen.org/privilege/vpnservice` | public |  | The application can manage the Virtual Private Network (VPN) and change its settings. |
 | `http://tizen.org/privilege/web-history.admin` | platform | User history | The application can manage user's internet history. |
 | `http://tizen.org/privilege/widget.viewer` | public |  | The application can show widgets, and information from their associated applications, on the home screen. |
 | `http://tizen.org/privilege/wifidirect` | public |  | The application can enable and disable Wi-Fi Direct, manage Wi-Fi Direct connections, and change Wi-Fi Direct settings. |
 | `http://tizen.org/privilege/window.priority.set` | public |  | The application can appear on top of other windows and screens, including the lock screen, according to the order of priority of the windows. This may prevent user from interacting with other applications or screens until the window for this application is closed. |
+| `http://tizen.org/privilege/windowsystem.admin` | platform | The application can change the settings for services provided by the display server, such as the quick panel and softkey bar. |
```
Comment (mijin85cho) [inline docs/application/native/tutorials/details/sec-privileges.md]: The last cell should be moved to right.

image

File: docs/application/dotnet/tutorials/sec-privileges.md [hunk 3]
```diff
@@ -78,6 +79,8 @@ when using security-sensitive API modules in .net applications:
 | `http://tizen.org/privilege/contact.read` | public | Contacts | The application can read user's profile, contacts, and contact history. Contact history can include social network activity. |
 | `http://tizen.org/privilege/contact.write` | public | Contacts | The application can create, update, and delete user's profile, contacts, and any contact history that is related to this application. Contact history can include social network activity. |
 | `http://tizen.org/privilege/content.write` | public |  | The application can change media information. This information can be used by other applications. |
+| `http://tizen.org/privilege/d2d.datasharing` | public | | The application can change the settings for services provided by the display server, such as the quick panel and softkey bar. |
```
Comment (mijin85cho) [inline docs/application/dotnet/tutorials/sec-privileges.md]: "The application can change the settings for services provided by the display server, such as the quick panel and softkey bar."

Is this sentence really for "http://tizen.org/privilege/d2d.datasharing"?

File: docs/application/native/tutorials/details/sec-privileges.md [hunk 4]
```diff
@@ -83,6 +84,8 @@ when using security-sensitive API modules in native applications:
 | `http://tizen.org/privilege/contact.read` | public | Contacts | The application can read user's profile, contacts, and contact history. Contact history can include social network activity. |
 | `http://tizen.org/privilege/contact.write` | public | Contacts | The application can create, update, and delete user's profile, contacts, and any contact history that is related to this application. Contact history can include social network activity. |
 | `http://tizen.org/privilege/content.write` | public |  | The application can change media information. This information can be used by other applications. |
+| `http://tizen.org/privilege/d2d.datasharing` | public | | The application can change the settings for services provided by the display server, such as the quick panel and softkey bar. |
```
Comment (mijin85cho) [inline docs/application/native/tutorials/details/sec-privileges.md]: "The application can change the settings for services provided by the display server, such as the quick panel and softkey bar."

Is this sentence really for "http://tizen.org/privilege/d2d.datasharing"?

File: docs/application/dotnet/tutorials/sec-privileges.md [hunk 5]
```diff
@@ -78,6 +79,8 @@ when using security-sensitive API modules in .net applications:
...
+| `http://tizen.org/privilege/d2d.remotelaunch` | public | | The application can be opened or used by apps on other devices. |
```
Comment (mijin85cho) [inline docs/application/dotnet/tutorials/sec-privileges.md]: apps -> applications

File: docs/application/native/tutorials/details/sec-privileges.md [hunk 6]
```diff
@@ -83,6 +84,8 @@ when using security-sensitive API modules in native applications:
...
+| `http://tizen.org/privilege/d2d.remotelaunch` | public | | The application can be opened or used by apps on other devices. |
```
Comment (mijin85cho) [inline docs/application/native/tutorials/details/sec-privileges.md]: apps -> applications

File: docs/application/dotnet/tutorials/sec-privileges.md [hunk 7]
```diff
@@ -140,24 +143,27 @@ when using security-sensitive API modules in .net applications:
...
+| `http://tizen.org/privilege/telephony` | public |  | The application can retrieve telephony information, such as the network and SIM card used and the statuses of calls. |
 | `http://tizen.org/privilege/telephony.admin` | platform |  | The application can manage telephony settings, such as those for incoming and outgoing calls, forwarding and holding calls, networks, and SIM cards. |
 | `http://tizen.org/privilege/tethering.admin` | platform |  | The application can enable and disable tethering services. This may result in additional charges depending on user's payment plan. |
 | `http://tizen.org/privilege/use_ir` | public |  | The application can use the infrared transmitter. |
 | `http://tizen.org/privilege/voicecontrol.manager` | platform |  | The application can record user's voice and recognize it so that voice commands can be used to control this app. It can also give responses to user's commands. |
+| `http://tizen.org/privilege/voicecontrol.tts` | partner |  | The application can use the voice control engine to recognize users' voice commands and provide voice feedback using its own voice. |
```
Comment (surbhig12) [inline docs/application/dotnet/tutorials/sec-privileges.md]: Suggestion, please change "users" to "you"

The application can use the voice control engine to recognize your voice commands and provide voice feedback using its own voice.

File: docs/application/native/tutorials/details/sec-privileges.md [hunk 8]
```diff
@@ -145,24 +148,27 @@ when using security-sensitive API modules in native applications:
...
+| `http://tizen.org/privilege/telephony` | public |  | The application can retrieve telephony information, such as the network and SIM card used and the statuses of calls. |
 | `http://tizen.org/privilege/telephony.admin` | platform |  | The application can manage telephony settings, such as those for incoming and outgoing calls, forwarding and holding calls, networks, and SIM cards. |
 | `http://tizen.org/privilege/tethering.admin` | platform |  | The application can enable and disable tethering services. This may result in additional charges depending on user's payment plan. |
 | `http://tizen.org/privilege/use_ir` | public |  | The application can use the infrared transmitter. |
 | `http://tizen.org/privilege/voicecontrol.manager` | platform |  | The application can record user's voice and recognize it so that voice commands can be used to control this app. It can also give responses to user's commands. |
+| `http://tizen.org/privilege/voicecontrol.tts` | partner |  | The application can use the voice control engine to recognize users' voice commands and provide voice feedback using its own voice. |
```
Comment (surbhig12) [inline docs/application/native/tutorials/details/sec-privileges.md]: Suggestion, please change "users" to "you"

The application can use the voice control engine to recognize your voice commands and provide voice feedback using its own voice.

File: docs/application/dotnet/tutorials/sec-privileges.md (hunk 1 above)
Comment (yl33) [reply docs/application/dotnet/tutorials/sec-privileges.md]: Done.

File: docs/application/native/tutorials/details/sec-privileges.md (hunk 2 above)
Comment (yl33) [reply docs/application/native/tutorials/details/sec-privileges.md]: Done.

File: docs/application/dotnet/tutorials/sec-privileges.md (hunk 3 above)
Comment (yl33) [reply docs/application/dotnet/tutorials/sec-privileges.md]: Thank you. Fixed.

File: docs/application/native/tutorials/details/sec-privileges.md (hunk 4 above)
Comment (yl33) [reply docs/application/native/tutorials/details/sec-privileges.md]: Thank you. Fixed.

File: docs/application/dotnet/tutorials/sec-privileges.md (hunk 5 above)
Comment (yl33) [reply docs/application/dotnet/tutorials/sec-privileges.md]: Done.

File: docs/application/native/tutorials/details/sec-privileges.md (hunk 6 above)
Comment (yl33) [reply docs/application/native/tutorials/details/sec-privileges.md]: Done.

File: docs/application/dotnet/tutorials/sec-privileges.md [hunk 9]
```diff
@@ -140,24 +143,27 @@ when using security-sensitive API modules in .net applications:
 | `http://tizen.org/privilege/recorder` | public | Microphone | The application can record video and audio. |
 | `http://tizen.org/privilege/screenshot` | platform |  | The application can capture screenshots. |
 | `http://tizen.org/privilege/secureelement` | public |  | The application can access secure smart card chips such as UICC/SIM, embedded secure elements, and secure SD cards. |
+| `http://tizen.org/privilege/securesysteminfo` | partner | Device unique ID | The application can read non-resettable secure device information. |
```
[the remark this answers is not in the document]
Comment (yl33) [reply docs/application/dotnet/tutorials/sec-privileges.md]: Done.

File: docs/application/dotnet/tutorials/sec-privileges.md [hunk 10]
```diff
@@ -140,24 +143,27 @@ when using security-sensitive API modules in .net applications:
...
 | `http://tizen.org/privilege/softap.admin` | platform |  | The application can turn SoftAP on or off, and change its settings. |
 | `http://tizen.org/privilege/systemmonitor` | public |  | The application can read system information, including information from the CPU and RAM. |
 | `http://tizen.org/privilege/systemsettings` | public |  | The application can read and write unrestricted system settings. Deprecated since 2.3.1. |
 | `http://tizen.org/privilege/systemsettings.admin` | platform |  | The application can read and write all system settings. |
 | `http://tizen.org/privilege/tee.client` | partner |  | The application can call security related functions running inside a Trusted Execution Environment (TEE), which ensures that sensitive data is stored, processed, and protected in an isolated, trusted environment. |
-| `http://tizen.org/privilege/telephony` | public |  | The application can retrieve telephony information, such as the network and SIM card used, the IMEI, and the statuses of calls. |
+| `http://tizen.org/privilege/telephony` | public |  | The application can retrieve telephony information, such as the network and SIM card used and the statuses of calls. |
```
[the remark this answers is not in the document]
Comment (yl33) [reply docs/application/dotnet/tutorials/sec-privileges.md]: Done.

File: docs/application/native/tutorials/details/sec-privileges.md [hunk 11]
```diff
@@ -145,24 +148,27 @@ when using security-sensitive API modules in native applications:
 | `http://tizen.org/privilege/recorder` | public | Microphone | The application can record video and audio. |
 | `http://tizen.org/privilege/screenshot` | platform |  | The application can capture screenshots. |
 | `http://tizen.org/privilege/secureelement` | public |  | The application can access secure smart card chips such as UICC/SIM, embedded secure elements, and secure SD cards. |
+| `http://tizen.org/privilege/securesysteminfo` | partner | Device unique ID | The application can read non-resettable secure device information. |
```
[the remark this answers is not in the document]
Comment (yl33) [reply docs/application/native/tutorials/details/sec-privileges.md]: Done.

File: docs/application/native/tutorials/details/sec-privileges.md [hunk 12]
```diff
@@ -145,24 +148,27 @@ when using security-sensitive API modules in native applications:
...
 | `http://tizen.org/privilege/softap.admin` | platform |  | The application can turn SoftAP on or off, and change its settings. |
 | `http://tizen.org/privilege/systemmonitor` | public |  | The application can read system information, including information from the CPU and RAM. |
 | `http://tizen.org/privilege/systemsettings` | public |  | The application can read and write unrestricted system settings. Deprecated since 2.3.1. |
 | `http://tizen.org/privilege/systemsettings.admin` | platform |  | The application can read and write all system settings. |
 | `http://tizen.org/privilege/tee.client` | partner |  | The application can call security related functions running inside a Trusted Execution Environment (TEE), which ensures that sensitive data is stored, processed, and protected in an isolated, trusted environment. |
-| `http://tizen.org/privilege/telephony` | public |  | The application can retrieve telephony information, such as the network and SIM card used, the IMEI, and the statuses of calls. |
+| `http://tizen.org/privilege/telephony` | public |  | The application can retrieve telephony information, such as the network and SIM card used and the statuses of calls. |
```
[the remark this answers is not in the document]
Comment (yl33) [reply docs/application/native/tutorials/details/sec-privileges.md]: Done.

File: docs/application/dotnet/tutorials/sec-privileges.md (hunk 7 above)
Comment (yl33) [reply docs/application/dotnet/tutorials/sec-privileges.md]: Done.

File: docs/application/native/tutorials/details/sec-privileges.md (hunk 8 above)
Comment (yl33) [reply docs/application/native/tutorials/details/sec-privileges.md]: Done.

File: docs/application/native/tutorials/details/sec-privileges.md [hunk 13]
```diff
@@ -69,6 +69,7 @@ when using security-sensitive API modules in native applications:
 | `http://tizen.org/privilege/appmanager.kill` | platform |  | The application can close other applications. |
 | `http://tizen.org/privilege/appmanager.kill.bgapp` | public |  | The application can request to close applications running in the background. |
 | `http://tizen.org/privilege/appmanager.launch` | public |  | The application can open other applications. |
+| `http://tizen.org/privilege/autofillmanager` | platform |  | The application can manage installed autofill services, detect which autofill service is currently being used, and change which autofill service to use. |
```
[the remark this answers is not in the document]
Comment (surbhig12) [reply docs/application/native/tutorials/details/sec-privileges.md]: @yl33,

Please verify this comment.

File: docs/application/dotnet/tutorials/sec-privileges.md [hunk 14]
```diff
@@ -64,6 +64,7 @@ when using security-sensitive API modules in .net applications:
 | `http://tizen.org/privilege/appmanager.kill` | platform |  | The application can close other applications. |
 | `http://tizen.org/privilege/appmanager.kill.bgapp` | public |  | The application can request to close applications running in the background. |
 | `http://tizen.org/privilege/appmanager.launch` | public |  | The application can open other applications. |
+| `http://tizen.org/privilege/autofillmanager` | platform |  | The application can manage installed autofill services, detect which autofill service is currently being used, and change which autofill service to use. |
```
[the remark this answers is not in the document]
Comment (yl33) [reply docs/application/dotnet/tutorials/sec-privileges.md]: I don't think it should be updated. Actually this sentence is already reviewed by STMS system.

[the remark this answers is not in the document]
Comment (yl33) [reply docs/application/dotnet/tutorials/sec-privileges.md]: I don't think it should be updated. Actually this sentence is already reviewed by STMS system.

[the remark this answers is not in the document]
Comment (surbhig12) [reply docs/application/dotnet/tutorials/sec-privileges.md]: @yl33,

Thank you for the response.

**Comment by mijin85cho** [comment]
Thanks for the check, @surbhig12 
@yl33 I have minor question.

`http://tizen.org/privilege/d2d.datasharing` for dotnet : The application can share data with other **devices**.
`http://tizen.org/privilege/d2d.datasharing` for native : The application can share data with other **applications**.

The same feature has slightly different sentence between dotnet and native.
(Anyway the meaning would be not that different in my thought.)
Is it intended?

**Comment by yl33** [comment]
Thank you. 'device' is correct. Fixed.

**Comment by yl33** [comment]
Sorry. I closed it by mistake. Reopened.

**Comment by mijin85cho** [comment]
Thanks for the update, @yl33 : )
As the change since @surbhig12's approval is minor (word change : applications -> devices),
I'll merge this PR without asking @surbhig12's re-approval.

Thanks for the review, @surbhig12.

**Pull request merged**